### ISSUE TITLE
Project forum

### ISSUE THREAD

**Issue by perkinslr** [rfk/pypyjs#119]
What are the odds of getting a forum for discussions not directly related to issues and pull requests?  It would be nice to be able to discuss things related to the project but which don't really belong on the issue tracker (showcases, questions and advice on setting it up, that sort of thing).

**Comment by rfk** [comment]
I'd be happy to wrangle something, do you have a concrete suggestion for hosting?  googlegroups or similar?

**Comment by perkinslr** [comment]
Not really, I've never done anything with googlegroups, but most of google's products are high quality, so it would probably work, and probably be reasonably spam-bot free.  The most concrete suggestion I would have is to use something like googlegroups rather than trying to manage a forum plugin on pypyjs.org, since I don't think we're likely to have much need of customization for it, and it removes most the headaches involved in administering the forum.

**Comment by rfk** [comment]
> rather than trying to manage a forum plugin on pypyjs.org

Doing this did not even enter my mind for a second :-)

**Comment by jedie** [comment]
A forum is IMHO a good idea! I found prefer to use a existing forum.

I'm a active member of http://www.python-forum.de Its the best python forum, IMHO... but this is a german forum ;) Sometimes english people ask a question, but his is rare...

If i search for "python forum" then the first English one is: http://www.python-forum.org/
Any experience?

**Comment by perkinslr** [comment]
While either of those would have the advantage of a (dedicated?) staff to eliminate lots of management headache, I think something specific to pypyjs would be a better thought.  It is possible that one of the major python forums would give us a dedicated subsection, which would work fine, but trying to intermingle pypyjs posts with normal python posts would defeat part of the advantage of having a forum.  sourceforge was nice enough to include forum and wiki pages when you host a project with them, but github doesn't, and I haven't found any common solution to that shortcoming.  I'd offer to just throw one up on my trac site, but my outbound internet pipe is already pretty well maxed.

**Comment by jedie** [comment]
Yes, sourceforge has many services. But i never seen a really active community there.

btw. django-cms used google+: https://plus.google.com/communities/107689498573071376044
But IMHO this doesn't work really good.

**Comment by perkinslr** [comment]
Yeah, the format of google+ doesn't really lend itself to organized community data.  I'm poking at groups.google.com, it looks simple and usable, but lacks any sort of tree organization, no sub forums.  Something similar to it but which supported subforums would work great.

**Comment by jedie** [comment]
btw. i logfed in my sourceforge account and see this: http://sourceforge.net/blog/tutorial-how-to-sync-a-github-or-google-code-repo-to-a-sourceforge-project/

**Comment by perkinslr** [comment]
That may be a good solution.  I've used sf in the past and their forums and what not are pretty good.  As for user communities around sf projects, it used to be more common, but most projects with an active community end up with a dedicated website, so forum or wiki pages on sf would be redundant.

**Comment by jedie** [comment]
I played a little bit in my sf.net account... It's all in all slow as in the past... The forum seems to be very elementary... IMHO not really a good idea ;)

**Comment by perkinslr** [comment]
Hm, yeah, their builtin ones are pretty basic.  Looks like the projects with good forums 'on' sourceforge are actually hosted elsewhere with a subdomain entry for <project>.sourceforge.net/forum.

**Comment by jedie** [comment]
A https://github.com/discourse/discourse solution is maybe nice. But didn't find a "free for opensource hoster"...

**Comment by perkinslr** [comment]
Wow, nice.  I don't know of any that let you do more than static html, and it looks kinda complicated to set up.  (basic setup is listed as half an hour.)

**Comment by Kebap** [comment]
> the first English one is: http://www.python-forum.org/
> Any experience?
I visit this forum for quite some years, and it has a nice helpful community as well.

**Comment by jedie** [comment]
I would suggest:

* just start on http://www.python-forum.org/ in sub-forum **Web Development** with prefix **[PyPyJS]**
* If enough posts come together: Ask for a separate sub-forum..
* if the community blows up: Setup a separate forum...

**Comment by kirbyfan64** [comment]
IMO, SourceForge's forums suck and are badly laid out. I have used Google Groups numerous times and only have good things to say about it.

**Comment by jedie** [comment]
I started a general conversation about PyPyJS here: http://www.python-forum.org/viewtopic.php?f=10&t=15769

**Comment by artyprog** [comment]
Hello @jedie, It would be nice to reactivate your Forum...
